presets/ragengine/vector_store/faiss_map_store.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+from typing import Any, List, cast
+
+from llama_index.core.schema import BaseNode
+from llama_index.core.bridge.pydantic import PrivateAttr
+from llama_index.vector_stores.faiss import FaissVectorStore
+
+class FaissVectorMapStore(FaissVectorStore):
```

Comment:
Oh wow this seems cool, looks like you found a way to actually get deletion/update to work with faiss! Legend. If this is experiencing issues, I would say though we can leave it unimplemented for faiss - faiss is a very simple little example other integrations may support this better

Reply:
I'm hopeful to get that going after this gets in!